pkg/csi/service/wcp/controller.go

```diff
@@ -137,7 +136,7 @@ func (c *controller) Init(config *cnsconfig.Config, version string) error {
 					return
 				}
 				log.Debugf("fsnotify event: %q", event.String())
-				if event.Op&fsnotify.Remove == fsnotify.Remove {
+				if event.Op&fsnotify.Remove == fsnotify.Remove && event.Name != cnsconfig.SupervisorCAFilePath {
```

Comment:
Can this be more specific, ie `event.Name == <name of cfgPath change event>`?

Reply:
Done